next/components/forms/simple-components/AccordionPaymentSchedule.tsx

```diff
@@ -0,0 +1,204 @@
+          <Button
+            variant="black-outline"
+            text="Pridať termíny do kalendára"
+            startIcon={<CalendarSchedule className="w-6 h-6" />}
+            className="lg:w-max w-full"
+          />
+        </div>
+        <div className="flex flex-col items-start p-6 lg:gap-6 gap-4 w-full bg-gray-50 rounded-lg">
+          <div id="content" className="flex lg:flex-row flex-col items-start lg:gap-6 gap-3 w-full">
+            <div className="grow items-start">
+              {t('payment_schedule.first_piece')}{' '}
+              <div className="text-h5 inline">{t('payment_schedule.first_piece_to')}</div>
+              {t('validity_decision')}
+            </div>
+            <div className="text-h5">29,66 €</div>
+          </div>
+          <div id="divider" className="w-full h-0.5 bg-gray-200" />
+          <div id="content" className="flex lg:flex-row flex-col items-start lg:gap-6 gap-3 w-full">
+            <div className="grow items-start">
+              {t('payment_schedule.second_piece')}
+              <div className="text-h5 inline">{t('payment_schedule.second_piece_to')}</div>
+            </div>
+            <div className="text-h5">29,66 €</div>
+          </div>
+          <div id="divider" className="w-full h-0.5 bg-gray-200" />
+          <div id="content" className="flex lg:flex-row flex-col items-start lg:gap-6 gap-3 w-full">
+            <div className="grow items-start">
+              {t('payment_schedule.third_piece')}
+              <div className="text-h5 inline">{t('payment_schedule.third_piece_to')}</div>
+            </div>
+            <div className="text-h5">29,66 €</div>
+          </div>
+        </div>
+      </div>
+      <div className="gap-3 lg:gap-0 flex flex-col">
+        <div className="text-h5">{t('payment_schedule.pay_with_qr')}</div>
+        <div className="text-h5-normal">{t('payment_schedule.change_amount')}</div>
+      </div>
+    </div>
+  )
+}
+
+const AccordionPaymentSchedule = ({
+  title,
+  size = 'sm',
+  icon = false,
+  shadow = false,
+  className,
+}: AccordionBase) => {
+  const [isActive, setIsActive] = useState(false)
+
+  const accordionSize = isAccordionSizeType(size) as AccordionSizeType
+
+  const paddingStyles = cx({
+    'px-4 py-3 lg:p-4': accordionSize === 'xs',
+    'p-4 lg:p-5': accordionSize === 'sm',
+    'p-4 lg:py-6 lg:px-8': accordionSize === 'md',
+    'py-5 px-6 lg:py-8 lg:px-10': accordionSize === 'lg',
+  })
+
+  const accordionHeaderStyle = cx(
+    'flex flex-col gap-4 w-full rounded-xl bg-gray-0',
+    className,
+    paddingStyles,
+  )
+  const accordionContainerStyle = cx(
+    'border-gray-200 flex flex-col w-full rounded-xl bg-gray-0',
+    className,
+    {
+      'border-2 border-solid hover:border-gray-500': !shadow,
+      'hover:shadow-[0_8px_16px_0_rgba(0,0,0,0.08)]': shadow,
+      'shadow-[0_0_16px_0_rgba(0,0,0,0.08)]': isActive && shadow,
+      'shadow-[0_4px_16px_0_rgba(0,0,0,0.08)]': !isActive && shadow,
```

Comment:
can you point me to where this styling exists in figma ? it does not seem right and I could not find it, the shadow prop also does not seem to be used